features/step_definitions/basic_steps.rb

```diff
@@ -22,8 +22,12 @@
   expect(current_path).to eq root_path
 end
 
+Then("show me the page") do
+  save_and_open_page
+end
+
```

Comment:
Remember to remove the show me the page as the last step :)

Reply:
Will do